measurements/pages/__init__.py

```diff
@@ -204,34 +208,69 @@ def files_in_country(country_code):
                            order_by=order_by,
                            current_country=country_code)
 
-
-@pages_blueprint.route('/files/download/<filename>')
-def files_download(filename):
-    try:
-        report = current_app.db_session.query(Report) \
-                        .filter(Report.textname.like("%" + filename)).first()
-        # XXX
-        # We have duplicate measurements :(
-        # So the below exception actually happens. This should be resolved
-        # in the data processing pipeline.
-        #   Report.filename == filename).one()
-    except NoResultFound:
-        raise NotFound("No file with that filename found")
-    except MultipleResultsFound:
-        # This should never happen.
-        raise HTTPException("Duplicate measurement detected")
-
-    if report is None:
+def decompress_autoclaved(autoclaved_filename, frames_off, frames_size, intra_off, intra_size):
+    def generator():
+        url = urljoin(current_app.config['AUTOCLAVED_BASE_URL'], autoclaved_filename)
+
+        current_app.logger.debug("Fetching autoclaved from: %s" % url)
+        current_app.logger.debug("  frames_off: %d" % frames_off)
+        current_app.logger.debug("  frames_size: %d" % frames_size)
+        current_app.logger.debug("  intra_off: %d" % intra_off)
+        current_app.logger.debug("  intra_size: %d" % intra_size)
+        try:
+            r = requests.get(url, headers={"Range": "bytes={}-{}".format(frames_off, frames_off + frames_size)},
+                                stream=True)
+            streamed_data = 0
+            for chunk in lz4framed.Decompressor(r.raw):
+                current_app.logger.debug("chunking away 1")
```

Comment:
mmm, don't understand the /meaning/ of this log message

Reply:
oops. These are some debug lines I left over. Will delete.